### ISSUE TITLE
n the training process, the center loss becomes larger and larger. What is the mistake

### ISSUE THREAD

**Issue by alanMachineLeraning** [davidsandberg/facenet#823]
![2](https://user-images.githubusercontent.com/17872637/43300033-96fceb70-918f-11e8-85bf-4e083779ccdc.png)

**Comment by fisakhan** [comment]
My be your training dataset is too small.

**Comment by jjsjunior** [comment]
Are you shuffling your trainset? I had this problem and after I starting shuffling the issue was resolved.

**Comment by RyanCV** [comment]
@jjsjunior do you mean shuffle here: `index_queue = tf.train.range_input_producer(range_size, num_epochs=None, shuffle=True, seed=None, capacity=32)`, I have tried "True" and "False", but it didn't help. 

@fisakhan : what do you mean by the training dataset is too small? I have 11k images (374 classes) for training, but I found the value in some part of the centers matrix (dimension is  374x128) become very large, any reason? Thanks.

**Comment by RyanCV** [comment]
@alanMachineLeraning How did you solve this problem of using center loss during training?  I also have this kind of inf of center loss after 3 epochs during finetuning of facenet.

**Comment by jetsmith** [comment]
@RyanCV @alanMachineLeraning i have the same problem, have you solved it? How?
